larq_compute_engine/tflite/python/interpreter_wrapper_utils.h

```diff
@@ -0,0 +1,201 @@
+#ifndef COMPUTE_ENGINE_TFLITE_PYTHON_INTERPRETER_WRAPPER_UTILS_H_
+#define COMPUTE_ENGINE_TFLITE_PYTHON_INTERPRETER_WRAPPER_UTILS_H_
+
+#include <cstring>
+#include <sstream>
+
+#include "pybind11/numpy.h"
+#include "pybind11/pybind11.h"
+#include "pybind11/stl.h"
+#include "tensorflow/lite/c/common.h"
+
+#define PY_ERROR(x)                                                           \
+  {                                                                           \
+    std::stringstream ss;                                                     \
+    ss << "ERROR at " << __FILE__ << ":" << __LINE__ << " : " x << std::endl; \
```

Comment:
(Don't apply the suggestion through the Github UI because it'll fail linting)
```suggestion
    ss << "ERROR at " << __FILE__ << ":" << __LINE__ << " : " << x << std::endl; \
```
This worked before because in C/C++ there is automatic concatenation of string literals, i.e `"ab" "c"` is equivalent to `"abc"`, and this macro was only used with `x` being a string literal. But it would be better to change it I think.

Reply:
Applied in 60b780fbb08a3902a77dcaaa3f85429bd771f405